doc/manual/getting_started_mgmt.rst

```diff
@@ -297,6 +305,8 @@ Note that the client key and certificate *must* match the client certificate giv
 
     To operate additional controllers and NDSPs over SSL, see :ref:`ctlrs-ssl`.
 
+    To connect with multiple clients and/or controller managers, you may concatenate the client certificate files into a single ``.pem`` file and use that for all connections. See `Sipyco SSL document <https://github.com/m-labs/sipyco/blob/master/doc/index.rst#ssl-setup>`_.
```

Comment:
The link actually doesn't say anything about "concatenation". And I'd keep the ARTIQ manual self-contained as much as possible.